lms/djangoapps/certificates/views.py

```diff
@@ -224,3 +232,227 @@ def update_example_certificate(request):
+        org=accd_course_org_html,
+        name=accd_company_name_html
+    )
+
+    # Translators:
+    context['accomplishment_more_title'] = _("More Information About {user_name}'s Certificate:").format(
+        user_name=user_fullname
+    )
+
+    # Translators:
+    context['certificate_date_issued_title'] = _("Issued On:")
+
+    # Translators:
+    context['certificate_date_issued'] = _('{month} {day}, {year}').format(
+        month=certificate.modified_date.strftime("%B"),
+        day=certificate.modified_date.day,
+        year=certificate.modified_date.year
+    )
+
+    # Translators:
+    context['certificate_id_number_title'] = _('Certificate ID Number')
+
+    # Translators:
+    context['certificate_info_title'] = _('About {company_name} Certificates').format(
+        company_name=company_name
+    )
+
+    cert_info_honor_code_html = "<a href='{tos_url}'>{company_name} Honor Code</a>".format(
+        tos_url=context.get('company_tos_url'),
+        company_name=company_name
+    )
+    cert_info_verified_html = "<a href='{verified_cert_url}'>verifying your identity</a>".format(
+        verified_cert_url=context.get('company_verified_certificate_url')
+    )
+    # Translators:
+    context['certificate_info_description'] = _("{company_name} acknowledges achievements through certificates, which "
+                                                "are awarded for various activities {company_name} students complete "
+                                                "under the {honor_code_html}.  Some certificates require completing "
+                                                "additional steps, such as {verifying_html}.").format(
+        company_name=company_name,
+        honor_code_html=cert_info_honor_code_html,
+        verifying_html=cert_info_verified_html
+    )
+
+    # Translators:
+    context['certificate_type_title'] = _('{certificate_type} Certfificate').format(
+        certificate_type=context.get('certificate_type')
+    )
+
+    # Translators:
+    context['certificate_verify_title'] = _("How {company_name} Validates Student Certificates").format(
+        company_name=company_name
+    )
+
+    # Translators:
+    context['certificate_verify_description'] = _("Certificates issued by {company_name} are signed by a gpg key so "
+                                                  "that they can be validated independently by anyone with the "
+                                                  "{company_name} public key. For independent verification, "
+                                                  "{company_name} uses what is called a "
+                                                  "&quot;detached signature&quot;.").format(company_name=company_name)
+
+    # Translators:
+    context['certificate_verify_urltext'] = _("Validate this certificate for yourself")
+
+    # Translators:
+    context['company_about_description'] = _("{company_name} offers interactive online classes and MOOCs from the "
```

Comment:
We have this string defined verbatim elsewhere in the platform. If you can find that string and EXACTLY match the sytanx we use, then that's one less string that translators have to translate (because they've already done it!)